src/components/Editor/index.js

```diff
@@ -405,6 +413,11 @@ class Editor extends PureComponent {
 
     const sourceId = selectedSource ? selectedSource.get("id") : "";
     const line = lineAtHeight(this.state.editor, sourceId, event);
+
+    if (isEmptyLine(line - 1)) {
```

Comment:
I find it slightly concerning that `onGutterContextMenu`'s `lineAtHeight` returns one number higher than `onGutterClick`'s `line` argument.

![linenumbers](https://user-images.githubusercontent.com/46655/29479484-11499e66-8438-11e7-9bce-07963fb62a2a.gif)

...but that existed before this PR.